element/lib/README.md

```diff
@@ -0,0 +1,17 @@
+## Atomic weights source
+
```

Comment:
This seems perfect to me! Might be useful to have a bibtex entry that can be copied or downloaded for the source? It should be easy enough for someone to find this and source it properly, but having the bibtex can help a lot of the time.

Reply:
Great idea. Thanks!